<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioDevicesBase</class>
 <widget class="QMainWindow" name="AudioDevicesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Devices</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>765</width>
         <height>331</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QComboBox" name="deviceBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="modeLabel">
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="deviceLabel">
            <property name="text">
             <string>Device</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="modeBox">
            <item>
             <property name="text">
              <string>Input</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Output</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Properties</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="3" column="0">
                 <widget class="QLabel" name="channelLayoutLabel">
                  <property name="text">
                   <string>Channel Layout</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="samplingRatesLabel">
                  <property name="text">
                   <string>Sampling Rates:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLineEdit" name="samplingRatesField">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="channelNumberLabel">
                  <property name="text">
                   <string>Number of Channels</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QLineEdit" name="channelNumberField">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="sampleFormatField_2">
                  <property name="text">
                   <string>Sample Formats:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLineEdit" name="sampleFormatField">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QLineEdit" name="channelLayoutField">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="preferredFormatLabel">
                  <property name="acceptDrops">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>Preferred Format</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="2">
                 <widget class="QLineEdit" name="preferredFormatField">
                  <property name="acceptDrops">
                   <bool>true</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
